### ISSUE TITLE
Can not use an image to link to an external site

### ISSUE THREAD

**Issue by jcea** [getnikola/nikola#1536]
Looks like when an image is clickable, Nikola considers than it is a preview and try to load the URL in a jquery colorbox element.

I am doing:

```
.. image:: logo.png
   :align: center
   :width: 581
   :height: 125
   :target: `Python Madrid`_
   :alt: Logo de Python Madrid

(blah blab)

.. _Python Madrid: http://www.python-madrid.es/
```
I want to show a clickable image that when clicked takes me to http://www.python-madrid.es/.

What happens is that Nikola generates this code:

```
<a class="reference external image-reference" href="http://www.python-madrid.es/"><img alt="Logo de Python Madrid" class="align-center" src="logo.png" style="width: 581px; height: 125px;"></a>
```
Not bad so far. But then "image-reference" is hijacked by Javascript at the end of the template:

```
<script src="../assets/js/all-nocdn.js"></script><script src="../assets/js/colorbox-i18n/jquery.colorbox-es.js"></script><script>jQuery("a.image-reference").colorbox({rel:"gal",maxWidth:"100%",maxHeight:"100%",scalePhotos:true});</script
```
If I comment out that javascript code, everything works fine.

Looking at the "bootstrap3" templates, looks like there is no option to disable that javascript by configuration.

**Comment by ralsina** [comment]
Why wontfix @Kwpolska ? I am fairly sure I can fix it unless it's a bad idea :-)

**Comment by Kwpolska** [comment]
@ralsina you sounded as if you didn’t want to do it… I’m okay with this change, and you do not even need my aproval.

**Comment by ralsina** [comment]
@jcea can you check that #1579 fixes this?

**Comment by Kwpolska** [comment]
```
Uncaught Error: Syntax error, unrecognized expression: a.image-reference,!.islink
```

fix: 4d334f7
```js
$('a.image-reference:not(.islink)')
```

**Comment by ralsina** [comment]
Ouch. Or $('a.image-reference',!'.islink')
